src/sql-parser/src/ast/defs/ddl.rs

```diff
@@ -458,16 +458,33 @@ impl_display!(DbzMode);
 
 #[derive(Debug, Clone, PartialEq, Eq, Hash, EnumKind)]
 #[enum_kind(ConnectorType)]
+pub enum CreateConnector<T: AstInfo> {
+    KafkaBroker {
+        broker: String,
+        with_options: Vec<SqlOption<T>>,
+    },
+}
+
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub enum KafkaConnector {
+    Literal { broker: String },
+    Reference { connector: UnresolvedObjectName },
+}
+
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub struct KafkaSource {
+    pub broker: KafkaConnector,
```

Comment:
```suggestion
pub struct KafkaSourceConnector {
    pub connector: KafkaConnector,
```

Reply:
Do we want to have `KafkaSourceConnector` as a type in two different crates with somewhat different fields?